docs: rework vectorizer instructions (#279)
timescale/pgai: docs/vectorizer-quick-start.md

## Setup a local development environment

> We use a docker compose file to set up a development environment, it includes a:

Reviewer: ```suggestion
To set up a development environment for OpenAI, use a docker compose file that includes a:
```
